docs/en/user_guides/useful_tools.md

```diff
@@ -0,0 +1,36 @@
+# Useful tools
+
+We also provide tools for users.
+
+## Print full config
+
+We incorporate modular and inheritance design into our [config](config.md) system, which is convenient to conduct various experiments. If you wish to inspect the config file, you may run `python tools/misc/print_config.py /PATH/TO/CONFIG` to see the complete config.
```

Comment:
+ 'MMGeneration incorporates [config](config.md) mechanism to set parameters used for training and testing models. With our config mechanism, users can easily conduct extensive experiments without hard coding. If you wish to inspect ...........'